### PR TITLE
Replace isotope

### PR DESCRIPTION
**Pull request by mikeyrayvon**
replace Isotope with Shuffle.js

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -28,6 +28,9 @@ function scripts_and_styles_method() {
     // Enqueue youtube api
     wp_enqueue_script('yt-player-api', 'http://www.youtube.com/player_api', array(), false, true);
 
+    // Enqueue Shuffle.js
+    wp_enqueue_script('shuffle-js', 'https://cdnjs.cloudflare.com/ajax/libs/Shuffle/5.0.2/shuffle.min.js', '', '', true);
```
Comment (varffs) [inline functions.php]: This should be bower installed. only the yt player isn't on bower thats why it is an enqueue

**Review by varffs — CHANGES_REQUESTED** [review]
The filtering is now additive. Meaning if I filter by one taxonomy then I add another filter I get everything that fulfils each of the taxonomies, rather than everything that fulfils both taxonomies.

File: functions.php (hunk 1 above)
Comment (mikeyrayvon) [reply functions.php]: did you check to see if shufflejs was one bower or just assume that i hadn’t 🤔

Comment (varffs) [reply functions.php]: yh i checked

Comment (mikeyrayvon) [reply functions.php]: screen shot 2017-10-23 at 12 54 02 pm

Comment (mikeyrayvon) [reply functions.php]: do you want to just ignore that?

Comment (mikeyrayvon) [reply functions.php]: we can define the dependencies explicitly in `package.json` but idk seems a little sketchy.

Comment (mikeyrayvon) [reply functions.php]: if thats what you want then commit it

**Pull request merged**